src/transformers/models/idefics/configuration_idefics.py

```diff
@@ -255,6 +255,7 @@ def __init__(
         alphas_initializer_range=0.0,
         alpha_type="float",
         rms_norm_eps=1e-6,
+        layer_norm_eps=1e-5,
```

Comment:
Rather than adding a new config variable that's only used in TF code, can we just pass `1e-5` to the relevant layer norm layers in TF? Then we wouldn't need any config changes at all.